pyomo/solvers/plugins/solvers/gurobi_direct.py

```diff
@@ -326,50 +348,59 @@ def _add_constraint(self, con):
 
         if con._linear_canonical_form:
             gurobi_expr, referenced_vars = self._get_expr_from_pyomo_repn(
-                con.canonical_form(),
-                self._max_constraint_degree)
-        #elif isinstance(con, LinearCanonicalRepn):
+                con.canonical_form(), self._max_constraint_degree
+            )
+        # elif isinstance(con, LinearCanonicalRepn):
         #    gurobi_expr, referenced_vars = self._get_expr_from_pyomo_repn(
         #        con,
         #        self._max_constraint_degree)
         else:
             gurobi_expr, referenced_vars = self._get_expr_from_pyomo_expr(
-                con.body,
-                self._max_constraint_degree)
+                con.body, self._max_constraint_degree
+            )
 
         if con.has_lb():
             if not is_fixed(con.lower):
-                raise ValueError("Lower bound of constraint {0} "
-                                 "is not constant.".format(con))
+                raise ValueError(
+                    "Lower bound of constraint {0} " "is not constant.".format(con)
```

Comment:
There are a few error messages in this file with extra quotes that should be removed.